hoomd/hpmc/compute.py

```diff
@@ -222,8 +238,10 @@ class SDF(Compute):
     smaller ``xmax``. Check that :math:`\sum_k s(x_k) \cdot dx \approx 0.5`.
 
     Warning:
-        `SDF` does not compute correct pressures for simulations with
-        concave particles or enthalpic interactions.
+        `SDF` only considers negative volume perturbations, and therefore does
+        not compute the correct pressure in simulations where positive volume
+        perturbations may change the system's potential energy, e.g., systems of
+        concave particles or with non-monotonic enthalpic interactions.
```

Comment:
```suggestion
        concave particles or with non-monotonic enthalpic interactions.
```
I suggested the word non-monotonic but now realize this isn't general enough. "Concave" interactions would also need expansion moves (consider a multi-site patch model). What word would better describe this?

We also need a second warning box that the pressure is correct only for potentials with constant values and step discontinuities (like square well and Kern-Frenkel).

Reply:
I think "non-monotonic" still covers "concave" interactions, if I'm picturing the what you mean by concave correctly. Consider a concave patchy interaction: the pair energy could change from 0 to `u_ij ≠ 0` back to 0 as the pair separation changes. So even if `u_ij` is constant when the particles are interacting, the pair energy is a non-monotonic function of the particle separation.

Reply:
"monotonic" applies to an expansion or compression along a single direction. What word would better indicate that U_ij is monotonic in all directions?

Reply:
Spherically symmetric?

Reply:
That's too restrictive. Maybe "non-monotonic" is close enough. We'll need to revisit this with the next PR that adds expansion perturbations.